src/app/charity/charity.component.ts

```diff
@@ -13,17 +15,37 @@ export class CharityComponent implements OnInit {
   campaigns: CampaignSummary[];
 
   constructor(
+    private datePipe: DatePipe,
     private pageMeta: PageMetaService,
     private route: ActivatedRoute,
   ) {}
 
   ngOnInit(): void {
-    this.campaigns = this.route.snapshot.data.campaigns;
+    this.campaigns = this.route.snapshot.data.campaigns.sort((a: CampaignSummary, b: CampaignSummary) => {
+      return new Date(b.endDate).getTime() - new Date(a.endDate).getTime();
+    });
 
     this.pageMeta.setCommon(
       this.campaigns[0] ? `${this.campaigns[0].charity.name} Campaigns` : 'Campaigns Archive',
       'Archive of Big Give match funded campaigns',
       null,
     );
   }
+
+  getPercentageRaised(childCampaign: CampaignSummary) {
+    CampaignService.percentRaisedOfIndividualCampaign(childCampaign);
+  }
+
+  getRelevantDateAsStr(campaign: CampaignSummary) {
+    const date = CampaignService.getRelevantDate(campaign);
+    return date ? this.datePipe.transform(date, 'dd/MM/yyyy, HH:mm') : null;
+  }
+
+  isInFuture(campaign: CampaignSummary) {
+    return CampaignService.isInFuture(campaign);
+  }
+
+  isInPast(campaign: CampaignSummary) {
+    return CampaignService.isInPast(campaign);
+  }
```

Comment:
Again not a blocker but it looks like these can be written a bit more simply as:

```suggestion
  isInFuture = CampaignService.isInFuture;

  isInPast = CampaignService.isInPast;
```